src/feature/relay/dns.c

```diff
@@ -1486,7 +1491,11 @@ configure_nameservers(int force)
   if (evdns_base_count_nameservers(the_evdns_base) == 1) {
     SET("max-timeouts:", "1000000");
   } else {
-    SET("max-timeouts:", "10");
+    if (options->TestingTorNetwork) {
```

Comment:
This code makes libevent stop using a DNS server when there is a timeout.

Please don't change this code.

We want libevent to switch servers after a few timeouts, so that we only disable IPv6 DNS if all the configured DNS servers are failing.